### PR TITLE
thanks citrusman

### PR DESCRIPTION
**Pull request by Jordie0608**
quack;

### CODE REVIEW

File: code/world.dm [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /world
+	name = "quack?"
```
Comment (Iamgoofball) [inline code/world.dm]: quack.

**Comment by kevinz000** [comment]
quack

**Comment by francinum** [comment]
You're a quack.

**Comment by Jordie0608** [comment]
test over go home nerds

**Pull request closed without merging**